pages/login.test.js

```diff
@@ -1,21 +1,24 @@
 import { render, screen } from '@testing-library/react';
+import { act } from 'react-dom/test-utils';
 
 import { loginFormTestIds } from '@/components/forms/Login';
 import Login, { loginPageTestIds } from './login';
 
 describe('Login', () => {
-  beforeEach(() => {
-    render(<Login />);
-  });
-
-  it('renders a login headline', () => {
+  it('renders a login headline', async () => {
+    await act(async () => {
+      render(<Login />);
+    });
     const headlineText = screen.getByTestId(loginPageTestIds.headlineText);
 
     expect(headlineText).toBeVisible();
     expect(headlineText).toHaveTextContent('Sign in to Nimble');
   });
 
-  it('renders a login form', () => {
+  it('renders a login form', async () => {
+    await act(async () => {
+      render(<Login />);
+    });
```

Comment:
```suggestion
    });
    
```

Reply:
done in 191115e3b3c247e8f417ba60f0c6ca950251c98d